scripts/azure-dev/AzureDev.Workstation.psm1

```
@@ -0,0 +1,1611 @@
+      throw (
+        'The local machine name could not be determined. Pass -WorkstationName ' +
+        'with a stable name for this workstation.'
+      )
+    }
+  }
+  return ConvertTo-AzureDevAccessName `
+    -Value $value `
+    -Label 'Workstation name'
+}
+
+function Set-AzureDevPrivatePermissions {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Set-AzureDevPrivatePermissions' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Set-AzureDevPrivatePermissions'

(PSUseSingularNouns)